less/components/icons.less

```diff
@@ -16,3 +16,15 @@
 .icon-2x {
   .icon(2em)
 }
+
+
+#icons {
+  position: absolute;
+  width: 0;
+  height: 0;
+}
+
+
+.icon.icon-globe {
```

Comment:
I also feel this is too general. Try moving this into `mymine.less` to target that use of icon-globe specifically.